exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Alias.kt

```diff
@@ -111,7 +111,7 @@ class QueryAlias(val query: AbstractQuery<*>, val alias: String) : ColumnSet() {
 
     override infix fun crossJoin(otherTable: ColumnSet): Join = Join(this, otherTable, JoinType.CROSS)
 
-    private fun <T : Any?> Column<T>.clone() = Column<T>(table.alias(alias), name, columnType)
+    private fun <T> Column<T>.clone() = Column<T>(table.alias(alias), name, columnType)
```

Comment:
[The default upper bound (if there was none specified) is `Any?`.](https://kotlinlang.org/docs/generics.html#generic-constraints:~:text=The%20default%20upper%20bound%20(if%20there%20was%20none%20specified)%20is%20Any%3F)